PlayerSkeleton.java

```diff
@@ -22,7 +22,7 @@ private int tryPossibleMoves(State s, int[][] legalMoves, double[] weights) {
 		//Step 1. Apply each move (action) to get the change field (board configuration).
 		//Step 2. Apply Heuristic.evaluate(s) to get the heuristic score from the move.
 		//Step 3. Get the best move.
-		double bestScore = 0;
+		double bestScore = Double.NEGATIVE_INFINITY;
```

Comment:
Evaluations can return negative values (doesn't mean it's a bad move!) Hence initialized lower bound to negative infinity instead of 0